### ISSUE TITLE
render natural=gully or other tag similar using current natural=cliff style

### ISSUE THREAD

**Issue by d1g** [gravitystorm/openstreetmap-carto#2288]
natural=cliff was defined at wiki with close relation to natural rocks, but  natural=cliff was heavily tagged for rendering: 
http://www.openstreetmap.org/way/191570269
http://www.openstreetmap.org/way/191570270

By many users, everywhere. Personally I relate this because of the symbols in "карты генштаба" maps.

We got new tag(s) natural=gully/valley to indicate a non-rocky change in the elevation.

**Comment by mboeringa** [comment]
A cliff does not necessarily have to be solid rock. There are many cases of soft sediment erosion along coasts, with pretty giant and high "cliffs" being formed that would definitely not classify as gully or valley, e.g. along the coast of the UK. I think the usage you are showing is within the limits of acceptable tagging for this. Alternatively, if the "gully" is actually a man_made=cutting, using the man_made=embankment tag would be an option.

**Comment by dieterdreist** [comment]
sent from a phone

> Il giorno 18 ago 2016, alle ore 11:14, mboeringa <notifications@github.com> ha scritto:
> 
> Alternatively, if the "gully" is actually a man_made=cutting, using the man_made=embankment tag would be an option.


??? Why the embankment and not the cutting tag if it's a cutting?

**Comment by d1g** [comment]
> A cliff does not necessarily have to be solid rock. 

Agree. natural=cliff is about "almost vertical drop" (see wiki). Almost always you get them with solid formations (simply put, "rocks"). But with coastal erosion you can get natural=cliff (a vertical drop) out of everything, yes.

> Alternatively, if the "gully" is actually a man_made=cutting, using the man_made=embankment tag would be an option.

Gullies and valleys are different from "vertical drop" because... they are not always as steep as "cliffs", right?
Gullies and valleys are different from man_made=embankment because they are not man made, right?

http://taginfo.openstreetmap.org/tags/man_made=cutting has only 60 uses ATM and wasn't described at wiki.

I don't see "almost vertical drops" based on Bing/other imagery here:
http://www.openstreetmap.org/way/173655182
http://www.openstreetmap.org/way/368611288
http://www.openstreetmap.org/way/368623276

Yes, they are steep, but not "near vertical" (natural=cliff). They are natural (not man_made=embankment) and most likely natural=gully or natural=valley (see wiki for details)

**Comment by d1g** [comment]
Other option is to tag http://www.bgs.ac.uk/landslides/happisburgh.html with natural=earth_bank + earth_bank=сut_bank (a river bank, erosion http://wiki.openstreetmap.org/wiki/File:Earth_bank_cut_bank.jpg)

IMO single [generic natural=earth_bank](http://wiki.openstreetmap.org/wiki/RU:Tag:natural%3Dearth_bank) is better than multiple overly narrow tags: natural=gully or natural=valley etc.

**Comment by mboeringa** [comment]
> ??? Why the embankment and not the cutting tag if it's a cutting?

Actually, for all natural or man_made "embankments" and "cuttings" there is a kind of grey area **for very large and wide ones** (yes even man_made ones can be extremely wide and huge):

To some extent, these large natural cliffs / gullies represent both: it depends on your current position and perception, whether your would perceive them as an embankment or cutting: if you are "down there", you would probably classify them as an "embankment". If you are "on the top", you might be more persuaded to call it a "cutting"...

So to some extent, these terms are interchangeable (and especially for the large and wide ones).

Anyway, _only man_made=embankment is currently rendered by Carto AFAIK..._

> http://taginfo.openstreetmap.org/tags/man_made=cutting has only 60 uses ATM and wasn't described at wiki.

Yes, you are right, I referred to a marginal undocumented tag... Probably a bad choice.

> We got new tag(s) natural=gully/valley to indicate a change in the elevation without explicit relation to rocks.

I have no idea how natural=valley plays a role here: valleys should not be rendered with a "cliff" like symbol. Imagine all valleys in the Alps bordered with such a "cliff" symbol...

Gully would be more of an option for a cliff like rendering: it is much closer to the kind of geological structure you seem to refer too, and generally much smaller.

**Comment by dieterdreist** [comment]
sent from a phone

> Il giorno 18 ago 2016, alle ore 16:35, mboeringa <notifications@github.com> ha scritto:
> 
> Actually, for all natural or man_made "embankments" and "cuttings" there is a kind of grey area for very large and wide ones (yes even man_made ones can be extremely wide and huge):
> 
> To some extent, these large natural cliffs / gullies represent both: it depends on your current position and perception, whether your would perceive them as an embankment or cutting:
> 


a cutting means you take something away, an embankment will typically be something (earth) added, and will be sloped, i.e. compared with the natural landscape. If it's vertical its a retaining wall.

**Comment by mboeringa** [comment]
> a cutting means you take something away, an embankment will typically be something (earth) added,

Agree, but there are situations in the real world where this distinction is difficult to make, and the history of a landscape simply is not known. Just look at the massive amounts of earth having been moved around in many cities with a long history like your favourite Rome ;-)... There may be vertical displacements and additions or removals of 5-10 meters easily all over the place in many such cities.

**Comment by d1g** [comment]
> There may be vertical displacements and additions or removals of 5-10 meters easily all over the place in many such cities.

Publicly available elevation data is often not able to capture them yet.

I simply use note=* to warn about "uneven elevation". We should have a tag that is not limited to "vertical drop" (natural=cliff) nor about "man made objects" (man_made=embankment).

The best option I was able to find is natural=earth_bank, any better suggestions?

**Comment by mboeringa** [comment]
@dieterdreist and others...

As an example of how careful one needs to be judging landscapes without any knowledge of the historic context, and that the distinction between embankments and cuttings is not always so "clear-cut", here is just one nice example near my home town in the Netherlands.

Look at this first image. Surely, this is a classic example of a nice embankment? The short road going down to the right into the fields clearly shows the drop in elevation (which in reality is bigger than you can judge from this picture, and amounts to probably some 2-3 meters), and the main road is elevated over its entire length. They must have put down an embankment here!

![embankment1](https://cloud.githubusercontent.com/assets/7635726/17802448/9db1c1d8-65f0-11e6-8a51-8abbf8637d80.jpg)

This small shaded relief DEM excerpt seems to confirm it: multiple embankments in a low lying Dutch polder? Surely!:

![embankment2](https://cloud.githubusercontent.com/assets/7635726/17802452/a46b0462-65f0-11e6-9c4e-56c0cd0aa69c.jpg)

The aerial of the area leaves you guessing... no hints here:

![embankment3](https://cloud.githubusercontent.com/assets/7635726/17802456/a9ea139c-65f0-11e6-9459-c9c0e4adb62a.jpg)

But now look at this larger section of the hillshaded DEM, including the natural coastal dune landscape to the left. What in reality seemed a classic example of an embankment, turns out to be multiple roads at their original elevation, with the surrounding landscape full of giant cuttings, areas where two 2-4 meters of sand has been removed. These millions of tons of sand now lie buried beneath 17th to 20th century city quarters of Amsterdam! This practice of extraction of sand has been going on for centuries here in this landscape. It is in these coastal dune cuttings where the classic Dutch tulips and other flowers are grown that the Pope in Rome each year thanks the Dutch for during his Easter blessings...

![embankment4](https://cloud.githubusercontent.com/assets/7635726/17802459/b2305ad4-65f0-11e6-8b2e-dbfceea66c82.jpg)

**Comment by d1g** [comment]
So what tag we should use as basis, so it is independent of:
* cutting/embankment decision and
* natural/man made decision
?

Both Russian and English definitions of the natural=earth_bank are not limited to natural or man_made objects:
* "A steep earth bank is **an abrupt change in ground level which can be clearly distinguished from its surroundings**, e.g. large erosion gullies, scraps, steep grassy slopes"
* "**Крутой перегиб поверхности земли, однозначно выделяющийся из окружающей среды**. Овраги, грунтовые обрывы, а также не выражающиеся горизонталями крутые задернованные склоны."

**Comment by pnorman** [comment]
I feel this is getting too much into a discussion on how to tag rather than a discussion on OpenStreetMap Carto. For the former, please use the tagging@ list or another suitable forum.

**Comment by d1g** [comment]
@pnorman, well rendering was proposed [at wiki](http://wiki.openstreetmap.org/wiki/Tag:natural%3Dearth_bank):

![](http://wiki.openstreetmap.org/w/images/a/a7/Earth_bank_Maperitive_sample.png)
Side rule is not different from anything else (top is on the left): http://wiki.openstreetmap.org/wiki/Tag:natural%3Dearth_bank#How_to_map

We are talking about another synonym to natural=cliff/embankment, right?

**Comment by dieterdreist** [comment]
sent from a phone

> Il giorno 19 ago 2016, alle ore 09:49, mboeringa <notifications@github.com> ha scritto:
> 
> This practice of extraction of sand has been going on for centuries here in this landscape. It is in these coastal dune cuttings where the classic Dutch tulips and other flowers are grown that the Pope in Rome each year thanks the Dutch for during his Easter blessings...



thank you for this interesting example. Btw, I wouldn't call these "cuttings" but maybe open pit mining, or excavation, this is not a "cut" anymore if it is on huge areas like here.

I think the word that people are looking for here is "slope",i.e. somehow similar to a cliff, but less steep. I would not use the same signature for cliffs and slopes.

**Comment by mboeringa** [comment]
> Btw, I wouldn't call these "cuttings" but maybe open pit mining, or excavation, this is not a "cut" anymore if it is on huge areas like here.

Completely agree ;-), calling these features "cuttings" was a stretch... only for the sake of the argument and to illustrate the story behind it.

**Comment by VA00** [comment]
For those, who do not live in areas heavily transformed by gullies, photos below might help. This is vast, complicated and old gully, recently mapped (incorrectly) as cliff:
https://www.openstreetmap.org/way/642683480.

Obviously, earth bank is more suitable, but it do not render, giving false impression of regular, flat beech forest, very easy to traverse. In reality you can quiclkly get lost in best, or fell off in worst case. Fresh ,,cuttings'' are nearly vertical (usually after heavy thunderstorm). Old are not vertical, but still very steep. 

![gully0](https://user-images.githubusercontent.com/16524226/48298534-66d63180-e4bf-11e8-9fb4-eb56d52746d4.jpg)
![gully1](https://user-images.githubusercontent.com/16524226/48298535-66d63180-e4bf-11e8-95b8-d509dbd65863.jpg)
![gully2](https://user-images.githubusercontent.com/16524226/48298536-676ec800-e4bf-11e8-80c5-89672c7f5a68.jpg)
![gully3](https://user-images.githubusercontent.com/16524226/48298537-676ec800-e4bf-11e8-9dbf-a59984f3238e.jpg)
![gully4](https://user-images.githubusercontent.com/16524226/48298538-676ec800-e4bf-11e8-9ccc-bebd43521ec4.jpg)
![gully5](https://user-images.githubusercontent.com/16524226/48298539-676ec800-e4bf-11e8-82c9-475940df1185.jpg)

**Comment by kocio-pl** [comment]
Thanks for reminding about it. I guess now it makes sense, since the usage is visible (2275 uses) - would you like to prepare the code?

https://taginfo.openstreetmap.org/tags/natural=earth_bank

![taghistory 25](https://user-images.githubusercontent.com/5439713/48299738-a1e16080-e4d1-11e8-9a97-b3252c05dc04.png)

**Comment by jragusa** [comment]
Using the same pattern as `man_made=embankment` would be correct ?

**Comment by kocio-pl** [comment]
I don't know. I guess this is closer to natural=cliff. Maybe some pattern should be different or maybe just less dense for example? It's always better if the mapper sees more precise feedback, but it's not always easy to tell what is the difference between tags.

**Comment by imagico** [comment]
I think the idea is that natural=cliff is meant for a vertical or near vertical drop which essentially implies a solid material like rock.  Therefore some mappers felt it is less appropriate to use the same tag for erosion structures in softer material which are often not that steep.  Unfortunately the tag is now used for a relatively wide range of structures, both with active erosion resulting in a sharp edge and stabilized slopes, essentially any type of naturally occurring steep slope in soft ground originally produced by erosion.  But there are also >600 uses of earth_bank=* for further differentiation.

There is overlap in use of natural=cliff and natural=earth_bank.  The latter is in particular popular in Russia where structures like this:

https://www.openstreetmap.org/way/120547840

would most likely be tagged this way.

**Comment by jeisenbe** [comment]
What about rendering in the same style as man_made=embankment as proposed above?

**Comment by andreynovikov** [comment]
The problem is that lots of mappers use cliff to map gully or mound. You can look at the current map of St.Petersburg, Russia - it's full of cliffs on the map. But it is completely flat. Flat! Yes, it's incorrect mapping and I can change tagging. But it will last for several days, no more. Poor mappers will come and turn everything back just because they do not see them on map.

**Comment by jeisenbe** [comment]
Are the `natural=earth_bank` features in St. Petersburg steep like cliffs or more like embankments?

**Comment by andreynovikov** [comment]
They are exactly like those on photos in https://github.com/gravitystorm/openstreetmap-carto/issues/2288#issuecomment-437564365

**Comment by andreynovikov** [comment]
Some examples:

https://www.openstreetmap.org/way/132189112
https://yandex.ru/maps/-/CGXuyB1k

https://www.openstreetmap.org/way/132189116
https://yandex.ru/maps/-/CGXuyAKN

The most stupid one:
https://www.openstreetmap.org/way/242404126
https://yandex.ru/maps/-/CGXu5Z4N

**Comment by Woazboat** [comment]
How about making `natural=earth_bank` a little bit more brownish and using more rounded 'spikes' on the rendered line to differentiate it from `natural=cliff`? I don't think they should both use the exact same style.

**Comment by GunSmoker** [comment]
I can confirm that I see natural=cliff everywhere instead of natural=earth_bank

This needs to be shown on the map, otherwise people will continue to use natural=cliff incorrectly.

**Comment by Francescotix** [comment]
I think that [the wiki entry](https://wiki.openstreetmap.org/wiki/Tag:natural%3Dearth_bank) provides an easy and practical distinction between "cliff" and "earth_bank":

> Unlike a natural=cliff, which is mainly rock, a natural=earth_bank is mainly comprised of soil or a mix of soil and stones.

So, in other words, a "cliff" is a rocky wall, which could be likely used for rock climbing, while an "earth_bank" is a slope, which may be more or less steep, more ore less rocky. 

Now, I'm not a geologist, but I would say that the "earth_bank" is carved by streams and rivers, while the "cliff" is the result of the movements of the Earth's crust. 
Consequently, I suggest that "earth_bank" should be rendered as "cliff" but with semicircles instead of arrows, since it is the result of the erosion made by the streams. Alternatively, it could be rendered as "embankment" but in green, for its natural origin.

In any case, I believe that "earth_bank" should be rendered in a better way than a basic green line.

**Comment by richlv** [comment]
Did anybody get to a proposed patch here? I'm about to map a few slopes as cliffs for now :)

**Comment by GunSmoker** [comment]
Looks like something like `natural=earth_bank` + `made_made=embankment` is the only option at the moment? I mean, if you want this to be displayed, and don't want to map incorrectly as `natural=cliff`.

**Comment by richlv** [comment]
That's still slightly incorrect tagging, as it's not man_made in that case.
For instances where I've wanted to get it rendered I've used natural=cliff. As we have very few cliffs in Latvia, could go over those later once earth_bank is rendered :)

**Comment by GunSmoker** [comment]
Yes, it is not man_made, that's the point. You can do auto-convert `natural=earth_bank` + `made_made=embankment` => `natural=earth_bank` in the future, while you can't do that for `natural=cliff` => 'natural=earth_bank'.

**Comment by richlv** [comment]
A very nice hack, thank you for the clarification :)
